### ISSUE TITLE
SIN numbers in export CSV button cause issues 

### ISSUE THREAD

**Issue by EricSchles** [18F/calc#1524]
1)
sometimes the data is exported like:

`871-1; 871-2 871-3; 871-4; 871-5; 871-6; 874-1; 874-4; 874-501; 874-503; 874-504; 874-505; 874-507;`

And other times it is exported like:

`"874-501, 874-501RC, 874-503, 874-503RC, 874-504, 874-504RC, 874-507, 874-507RC"`

And there appears to be no rhyme or reason for which or when.  So we should resolve this, so that one consistent format is shown.  

2)
This causes a larger problem so the CSV viewer doesn't show data in it's headers correctly.  Example

<img width="311" alt="screen shot 2017-03-27 at 4 08 12 pm" src="https://cloud.githubusercontent.com/assets/928957/24376273/4ca0a856-1309-11e7-9e05-d62fb8841fd8.png">

**Comment by jseppi** [comment]
Related #1033

**Comment by jseppi** [comment]
@EricSchles I just downloaded the CSV via the Export Data button and am not able to reproduce this problem in Excel. I also checked the SIN column cells in `less` -- they all look properly `"`-delimited. Is it possible that Office Libre is handling the file incorrectly?

Also, the code for this is in `api/views.py#GetRatesCSV`, which is using python's standard `csv` module to write the csv rows. Its defaults are configured to quote-delimit any cells that have comma-containing strings.
